### PR TITLE
Add two_sum function and tests

### PR DESCRIPTION
**Pull request by gpark439**

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
+            if n[j] == t - n[i]:
+                return [i, j]
```
Comment (NinaPatrina) [inline functions/two_sum.py:5]: nice job

File: functions/two_sum.py [hunk 2]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
```
Comment (NinaPatrina) [inline functions/two_sum.py:2]:
```suggestion
    for k in range(len(n)):
```

**Review by NinaPatrina — COMMENTED** [review]
Nice lob Grace. Please look at my suggestion

File: functions/two_sum.py (hunk 2 above)
Comment (gpark439) [reply functions/two_sum.py:2]: I IS BETTER THAN K NO THANK U

Comment (NinaPatrina) [reply functions/two_sum.py:2]: ok I it is

**Pull request merged**